content/exchange/artifacts/Generic.Remediation.UploadFile.Glob.yaml

```diff
@@ -0,0 +1,34 @@
+name: Generic.Remediation.UploadFile.Glob
+description: |
+   This artefact lets you upload a file to the target system. If the target file exists, it will be overridden.
+   It uses glob to define the destination path. You can upload the file to multiple destinations at once.
+
+type: CLIENT
+
+parameters:
+   - name: TargetGlob
+     description: Destination Path, if the destination already exists, it will be overridden
+   - name: ReallyDoIt
+     description: When selected, it will really override the targets!
+     type: bool
+     
+tools:
+  - name: RemediationFile
+    url:
+
+sources:
+  - query: |
+      LET targets = SELECT * FROM glob(globs=TargetGlob)
+        WHERE NOT if(condition=TRUE,
```

Comment:
This seems a bit pointless? Isnt this the same as `WHERE NOT IsDir`

Reply:
Yes, that was redundant, I fixed it in the newer commit.